TwitchLib.Client/Events/NoticeEventArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace TwitchLib.Client.Events;
+
+/// <summary>
+/// EventArgs representing a NOTICE.
+/// </summary>
+public class NoticeEventArgs
```

Comment:
I dont know how I feel about this.
Sure its a lot cleaner to maintain but you also loose the ability to choose what you want to be notified about and you basically dont know what you get with this Event.
Sure you got the message but that might change with time so relying on that to parse out what Twitch actually sent you isnt a future proof thing.
Maybe at least add the MsgId to the event?